### ISSUE TITLE
Some problem in training

### ISSUE THREAD

**Issue by Crocs512** [mcahny/Deep-Video-Inpainting#30]
Thank you very much for the given code, but I have a few questions about the details of the training.

After I have rewritten the training code according to the given training demo, I found that the results during the training process were vague and the loss was very unstable.

I used YouTube’s Vos dataset, I don’t know if it’s my implementation problem or what? So I look forward to the author can tell me more implementation details！！

Thanks！

**Comment by mcahny** [comment]
Hi,

For training, I would recommend you to do multi-stage training.
Also, in the training dataset, you will need to include easy samples such
as Figure3-(b) in the paper
<https://openaccess.thecvf.com/content_CVPR_2019/papers/Kim_Deep_Video_Inpainting_CVPR_2019_paper.pdf>
.

1st stage training is the 2-tower network (1 reference, 1 target) which is
the easiest stage.
2nd stage you may scale up to 5-tower network (1 ref, 4 tar),
3rd stage, you can put the recurrent feedback, 6-tower (1 ref, 5 ref) ,
where the last reference frame is the recurrent feedback.


Best,
Dahun




2021년 3월 19일 (금) 오전 11:33, Crocs512 ***@***.***>님이 작성:
